Cargo.toml

```diff
@@ -20,10 +16,16 @@ license = "MIT"
 keywords = [ "win32", "outlook", "mapi" ]
 categories = [ "os::windows-apis" ]
 
-[workspace.dependencies]
-outlook-mapi-stub = "0.3.1"
-outlook-mapi-sys = { version = "0.7.3", default-features = false }
+[workspace.dependencies.outlook-mapi-stub]
+version = "0.3.1"
+path = "crates/mapi-stub"
 
+[workspace.dependencies.outlook-mapi-sys]
+version = "0.7.4"
+path = "crates/mapi-sys"
```

Comment:
I noticed through trial and error that `release-plz update` seems to be able to bump the version of the dependencies when you include the `version` and `path` in the dependency directly, but not when using the `patch` table like this was. This multiline format is equivalent to putting all of that inside of `{...}` on a single line, but that single line was getting kind of long.